data/common.yaml

```diff
@@ -39,6 +39,7 @@ simp::scenario_map:
     - simp::admin
     - simp::sssd::client
     - ssh
+    - simp::scenario::poss
```

Comment:
@jeefberkey We can absolutely test this since it the base `simp` scenario. I'm fine with changing it to `one_shot` and we can support it but the first question will be "what did you change".